app/kinesis/ProductionMetricsIndexStreamReader.scala

```diff
@@ -40,7 +51,41 @@ object ProductionMetricsStreamReader {
 
     val codec = KinesisEvent
 
-    override protected def processEvent(eventWithSize: EventWithSize[KinesisEvent]): Unit =
-      println(s"Processed event ${eventWithSize.event}")
+    override protected def processEvent(eventWithSize: EventWithSize[KinesisEvent]): Unit = {
+      val event = eventWithSize.event
+      val eventType = event.eventType
+      eventType match {
+        case CapiContent => processCapiEvent(event.eventJson)
+        case _ => Logger.error(s"Invalid event type on kinesis stream could not be processed ${event}")
+      }
+    }
+
+    private def convertStringToDateTime(dateTime: String): Option[DateTime] = {
+      val formatter = DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
+      try {
+        Some(formatter.parseDateTime(dateTime))
+      }
+      catch {
+        case e: Throwable => {
+          Logger.error(s"String $dateTime could not be converted to datetime. $e")
+          None
+        }
+      }
+    }
+
+    private def processCapiEvent(json: Json) =
+      Parser.jsonToCapiData(json) match {
+        case Right(data) => putCapiDataInDB(data)
+        case Left(error) => Logger.error(error.message)
+      }
+
+
+    def putCapiDataInDB(data: CapiData) = {
+      for {
+        date <- convertStringToDateTime(data.creationDate)
+        metric = Metric(UUID.randomUUID().toString, data.originatingSystem, Some(data.composerId), data.storyBundleId,
+          Some(data.commissioningDesk), None, None, None, date, None)
```

Comment:
Not important, but it would be easier to read if you'd include the field names for the `None` fields.